### PR TITLE
Add tests for forms: account activation, account reset, password reset

### LINKED ISSUE #9: [F] E2E tests
- [x] account activation form (fail/success)
- [x] password reset form (fail/success)
- [x] account reset form (fail/success)

### PR DESCRIPTION
**Pull request by Karkavin**
resolve #9

### CODE REVIEW

**Comment by nikon-petr** [comment]
resolve #9

File: angio-frontend/tests/e2e/fixtures/modules/user/account_activation.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        "description" : "Просмотр анализа"
+      }, {
+        "id" : 11,
+        "name" : "TOKEN_REVOKE",
+        "description" : "Отзыв собственных refresh токенов"
+      } ]
+    } ],
+    "ownedRolesToManage" : [ ]
+  }
+}
```
[the remark this answers is not in the document]
Comment (Karkavin) [reply angio-frontend/tests/e2e/fixtures/modules/user/account_activation.json]: Done

File: angio-frontend/tests/e2e/fixtures/modules/user/reset_account.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "status" : "SUCCESS",
+  "date" : "2019-10-02T13:45:14Z"
+}
```
[the remark this answers is not in the document]
Comment (Karkavin) [reply angio-frontend/tests/e2e/fixtures/modules/user/reset_account.json]: Done

File: angio-frontend/tests/e2e/fixtures/modules/user/reset_password.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+  "status" : "SUCCESS",
+  "date" : "2019-10-02T13:45:14Z"
+}
```
[the remark this answers is not in the document]
Comment (Karkavin) [reply angio-frontend/tests/e2e/fixtures/modules/user/reset_password.json]: Done

File: angio-frontend/tests/e2e/specs/modules/user/resetPassword.ts [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+            .type('newUser@example.com');
+
+        cy.get('[data-test-id="submit__button"]')
+            .click();
+
+        cy.location('pathname').should('eq', '/user/reset');
+        cy.get('[data-test-id="formMsg__list"]')
+            .children().should('have.length', 1);
+    });
+});
```
[the remark this answers is not in the document]
Comment (Karkavin) [reply angio-frontend/tests/e2e/specs/modules/user/resetPassword.ts]: Done

File: angio-frontend/tests/e2e/specs/modules/user/resetAccount.ts [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+            .type('q1w2e3r4t5');
+
+        cy.get('[data-test-id="submit__button"]')
+            .click();
+
+        cy.location('pathname').should('eq', '/reset');
+        cy.get('[data-test-id="formMsg__list"]')
+            .children().should('have.length', 1);
+    });
+});
```
[the remark this answers is not in the document]
Comment (Karkavin) [reply angio-frontend/tests/e2e/specs/modules/user/resetAccount.ts]: Done

File: angio-frontend/tests/e2e/specs/modules/user/accountActivation.ts [hunk 6]
```diff
@@ -0,0 +1,67 @@
+describe('Login', () => {
```
[the remark this answers is not in the document]
Comment (Karkavin) [reply angio-frontend/tests/e2e/specs/modules/user/accountActivation.ts]: Done

File: angio-frontend/tests/e2e/fixtures/modules/user/reset_password.json [hunk 7]
```diff
@@ -0,0 +1,4 @@
+{
```
[the remark this answers is not in the document]
Comment (Karkavin) [reply angio-frontend/tests/e2e/fixtures/modules/user/reset_password.json]: Done

File: angio-frontend/tests/e2e/specs/modules/user/accountActivation.ts [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+
+        cy.location('pathname').should('eq', '/login');
+    });
+
+    it('Account activation (fail)', () => {
+
+        cy.server();
+        cy.route({
+            method: 'POST',
+            url: `${Cypress.env('CYPRESS_API_BASE_URL')}/api/v2/user/undefined/enable`,
```
[the remark this answers is not in the document]
Comment (Karkavin) [reply angio-frontend/tests/e2e/specs/modules/user/accountActivation.ts]: Yes, sure, we mocked it. We don't need it.

File: angio-frontend/src/modules/user/components/AccountActivationForm.vue [hunk 9]
```diff
@@ -7,12 +7,19 @@
...
+                            ref="form"
+                            id="activation-form"
+                            data-test-id="accountActivation__form"
+                    >
                         <v-text-field
                                 v-model="form.enablingCode"
                                 v-bind:label="$t('user.component.activation.activationFormStep.enablingCode.field')"
                                 v-bind:rules="[v => !!v || $t('user.component.activation.activationFormStep.enablingCode.validation.NotEmpty')]"
                                 v-bind:disabled="fetching"
+                                data-test-id="enablingCode_input"
```
[the remark this answers is not in the document]
Comment (Karkavin) [reply angio-frontend/src/modules/user/components/AccountActivationForm.vue]: Done

**Pull request merged**